src/ontology/cl-edit.owl

```diff
@@ -33756,6 +33764,17 @@ AnnotationAssertion(rdfs:label obo:CL_4052042 "tuft cell of urethra")
 EquivalentClasses(obo:CL_4052042 ObjectIntersectionOf(obo:CL_0002204 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002325)))
 SubClassOf(obo:CL_4052042 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_1904320))
 
+# Class: obo:CL_4052045 (steroidogenic stromal cell of ovary)
```

Comment:
For other reviewers: this change here, which is seemingly unrelated to the rest of the PR, is simply due to the fact that I re-serialized the -edit file as part of the conflict resolution process. That class had somehow been written out-of-order (after the CL:4052027 class) at some point in the master branch, so now the re-serialization put it back at the right place. Everything’s normal.